### PR TITLE
hwdb: Add Lenovo G580 

### LINKED ISSUE #23066: Incorrect touchpad dimensions on Lenovo g580
**systemd version the issue has been seen with**
 >`systemd 250 (250.4-1~bpo11+1)`

**Used distribution**
 > `Debian GNU/Linux 11 (bullseye)`

**Linux kernel version used** (`uname -a`)

 > `Linux deb 5.16.0-0.bpo.4-amd64 #1 SMP PREEMPT Debian 5.16.12-1~bpo11+1 (2022-03-08) x86_64 GNU/Linux`

**CPU architecture issue was seen on**
 > x86_64

**Expected behaviour you didn't see**
 > Touchpad is working correctly

**Unexpected behaviour you saw**
 > Incorrect touchpad dimensions

**Steps to reproduce the problem**
 > Laptop lenovo g580 and linux kernel 5

The touchpad problem is solved by creating /etc/udev/hwdb.d/61-evdev-local.hwdb and running commands `systemd-hwdb update;udevadm trigger /dev/input/event*` as instructed inhttps://github.com/systemd/systemd/blob/main/hwdb.d/60-evdev.hwdb
An entry in 61-evdev-local.hwdb with which my touchpad works correctly
```
#Lenovo g580
evdev:name:ETPS/2 Elantech Touchpad:dmi:*svnLENOVO:*pvrLenovoG580**
 EVDEV_ABS_00=-170:2950:24
 EVDEV_ABS_01=-591:700:25
 EVDEV_ABS_35=-170:2950:24
 EVDEV_ABS_36=-591:700:25
```

Without 61-evdev-local.hwdb
```
udevadm info /dev/input/event6
P: /devices/platform/i8042/serio1/input/input6/event6
N: input/event6
L: 0
S: input/by-path/platform-i8042-serio-1-event-mouse
E: DEVPATH=/devices/platform/i8042/serio1/input/input6/event6
E: DEVNAME=/dev/input/event6
E: MAJOR=13
E: MINOR=70
E: SUBSYSTEM=input
E: USEC_INITIALIZED=9392282
E: ID_INPUT=1
E: ID_INPUT_TOUCHPAD=1
E: ID_INPUT_WIDTH_MM=47
E: ID_INPUT_HEIGHT_MM=20
E: ID_BUS=i8042
E: ID_SERIAL=noserial
E: ID_PATH=platform-i8042-serio-1
E: ID_PATH_TAG=platform-i8042-serio-1
E: ID_INPUT_TOUCHPAD_INTEGRATION=internal
E: LIBINPUT_DEVICE_GROUP=11/2/e:isa0060/serio1
E: DEVLINKS=/dev/input/by-path/platform-i8042-serio-1-event-mouse
```

with 61-evdev-local.hwdb
```
P: /devices/platform/i8042/serio1/input/input6/event6
N: input/event6
L: 0
S: input/by-path/platform-i8042-serio-1-event-mouse
E: DEVPATH=/devices/platform/i8042/serio1/input/input6/event6
E: DEVNAME=/dev/input/event6
E: MAJOR=13
E: MINOR=70
E: SUBSYSTEM=input
E: USEC_INITIALIZED=9392282
E: EVDEV_ABS_00=0:3000:24
E: EVDEV_ABS_01=-591:630:23
E: EVDEV_ABS_35=-180:3000:24
E: EVDEV_ABS_36=-591:630:25
E: ID_INPUT=1
E: ID_INPUT_TOUCHPAD=1
E: ID_INPUT_WIDTH_MM=125
E: ID_INPUT_HEIGHT_MM=53
E: ID_BUS=i8042
E: ID_SERIAL=noserial
E: ID_PATH=platform-i8042-serio-1
E: ID_PATH_TAG=platform-i8042-serio-1
E: ID_INPUT_TOUCHPAD_INTEGRATION=internal
E: LIBINPUT_DEVICE_GROUP=11/2/e:isa0060/serio1
E: DEVLINKS=/dev/input/by-path/platform-i8042-serio-1-event-mouse
```

### PR DESCRIPTION
**Pull request by p-fpv**
Add rules for Lenovo G580.
Fixes #23066

### CODE REVIEW

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -622,6 +622,13 @@ evdev:name:MSFT0001:02 04F3:304B Touchpad:dmi:*svnLENOVO:*pvrLenovoLegionY9000X2
  EVDEV_ABS_35=::31
  EVDEV_ABS_36=::30
 
+#Lenovo g580
```
Comment (bluca) [inline hwdb.d/60-evdev.hwdb]: missing space

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
Why negative values used?
```
/home/runner/work/systemd/systemd/build/../hwdb.d/60-evdev.hwdb: 95 match groups, 124 matches, 346 properties
Failed to parse: 'EVDEV_ABS_00=-170:2950:24'
Failed to parse: 'EVDEV_ABS_01=-591:700:25'
Failed to parse: 'EVDEV_ABS_35=-170:2950:24'
Failed to parse: 'EVDEV_ABS_36=-591:700:25'
```

**Comment by yuwata** [comment]
Hm, looking the udev code, and linux/input.h, negative value seems to be allowed.
The parser needs to be updated.

**Comment by yuwata** [comment]
The parser is updated.

**Pull request merged**